apps/extension/src/ui/domains/Staking/Bittensor/hooks/useBittensorStakingPayload.ts

```diff
@@ -69,6 +75,38 @@ export const useBittensorStakingPayload = ({
     return taoToAlpha(minTaoStake, alphaPrice)
   }, [minTaoStake, alphaPrice])
 
+  // simulate swap for minimum stake to get swap fee
+  const { data: minStakeSwapSimulation } = useQuery({
+    queryKey: ["minStakeSwapSimulation", sapi, netuid, minTaoStake?.toString()],
```

Comment:
sapi object wont serialize consistently, we have a id property to workaround that

```suggestion
    queryKey: ["minStakeSwapSimulation", sapi?.id, netuid, minTaoStake?.toString()],
```